spec/rspec/github/formatter_spec.rb

```diff
@@ -137,4 +137,28 @@
       end
     end
   end
+
+  describe '#seed' do
+    before { formatter.seed(notification) }
+
+    context 'when seed used' do
+      let(:notification) do
+        RSpec::Core::Notifications::SeedNotification.new(4242, true)
+      end
+
+      it 'outputs the fully formatted seed notification' do
+        is_expected.to eq notification.fully_formatted
```

Comment:
This test uses the same code as the code itself. It now really depends on the output of the RSpec method. It might be nice to either change this spec or add a spec that checks against the output we actually want (which is information about the seed).

So adding a test that checks that the output is something like "Seed is 4242" or whatever it returns. It does add a dependency to the RSpec module but I actually think that's a good thing in this case since you want to ensure the actual seed information is outputted.

Reply:
Sure thing, updated.